week-2/implement/cases.js

```diff
@@ -15,3 +15,15 @@
 
 // Come up with a clear, simple name for the function
 // Use the string documentation to help you plan your solution
+
+
+function UpperSnakeCase(text){
```

Comment:
I have a small one here: it's recommended that you start with lowercase. Anything that's not a constructor usually starts with lowercase and is [camelCased](https://en.wikipedia.org/wiki/CamelCase).